frontend/src/App.css

```diff
@@ -1,7 +1,3 @@
 .pf-c-toolbar__content-section {
     row-gap: 16px !important;
 }
-
```

Comment:
Wondering about this change?

Reply:
Fix is available in version of PF console is using:
https://github.com/patternfly/patternfly/issues/4053